jdaviz/utils.py

```diff
@@ -1,42 +1,7 @@
-import os
 import time
 import threading
 from collections import deque
 
```

Comment:
Don't delete `__all__`, just keep it empty.
```suggestion

__all__ = []
```